src/utils.js

```diff
@@ -12,8 +12,9 @@ const run = cmd => {
       {
         stdio: [0, 'pipe', 'ignore'],
       },
-      (err, out) => {
-        resolve((err ? '' : out.toString() || '').trim());
+      (err, out, stderr) => {
+        const output = out.toString() + stderr.toString();
```

Comment:
I was trying to avoid doing this for every command.

Some commands need this, and other's don't. Can you think of any negative side effects?

Reply:
Not I can think of. It's a little weird that why some software put version info in the error stream.

Reply:
It's super weird and irritating. I believe Python does this too. 

I think I would feel better about this and possibilities of regression if outputting stderr was only used with a flag in this function - `const run = (cmd, options) => {`

And rename out to stdout

Reply:
Fixed.